v2/googlecloud-to-neo4j/src/test/java/com/google/cloud/teleport/v2/neo4j/templates/GoogleToNeo4jTest.java

```diff
@@ -71,13 +73,4 @@ public void testResolvedSqlVariable() {
     String uriReplaced = ModelUtils.replaceVariableTokens(uri, optionsParams.getTokenMap());
     assertThat(uriReplaced).contains("LIMIT 7");
   }
-
-  @Test
-  public void testGetInvalidOrderedQuery() {
```

Comment:
TODO: reinstate this?

Reply:
`InputValidator` is gone and this check is hard to implement properly, so let's ditch it for now